matplotcheck/tests/test_base_data.py

```diff
@@ -1,7 +1,9 @@
 """Tests for the base module -- Data"""
 import pytest
+from matplotcheck.base import PlotTester
 import matplotlib.pyplot as plt
 import numpy as np
+import pandas as pd
 import random
```

Comment:
@ryla5068 is the random package actually used in these modules? pycharm is suggesting that is it not.